pkg/hubble/peer/cell/cell.go

```diff
@@ -0,0 +1,104 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package cell
+
+import (
+	"fmt"
+	"log/slog"
+	"strconv"
+	"strings"
+
+	"github.com/cilium/hive/cell"
+
+	"github.com/cilium/cilium/pkg/hubble/peer"
+	"github.com/cilium/cilium/pkg/hubble/peer/serviceoption"
+	"github.com/cilium/cilium/pkg/logging/logfields"
+	nodeManager "github.com/cilium/cilium/pkg/node/manager"
+)
+
+// Cell provides the Hubble peer service that handles peer discovery and notifications.
+var Cell = cell.Module(
+	"hubble-peer-service",
+	"Hubble peer service for handling peer discovery and notifications",
+
+	cell.Provide(newPeerService),
+)
+
+// HubbleConfig interface defines the configuration needed by the peer service
+type HubbleConfig interface {
+	GetListenAddress() string
+	GetPreferIPv6() bool
+	IsTLSEnabled() bool
+}
```

Comment:
Question: Do we need an interface or could we make this work with a plain struct ? Feels like there is not much use here.